tests/agent_unittests/test_utilization_settings.py

```diff
@@ -118,7 +118,7 @@ def reset(wrapped, instance, args, kwargs):
     return reset
 
 
-@reset_agent_config(INI_FILE_WITHOUT_UTIL_CONF, ENV_WITHOUT_UTIL_CONF)
+@reset_agent_config(INI_FILE_WITHOUT_UTIL_CONF, {"NEW_RELIC_HOST": "collector.newrelic.com"})
```

Comment:
It's concerning to me that the HOST is needed here. It shouldn't need the host to default the OTLP_HOST. Can you explain more about this change/why this test was failing? I just tried running it locally and it didn't fail for me.

Reply:
This test is really annoying and fails if you've got NEW_RELIC_HOST set to point to staging. It causes my local instance to fail all the time.

If you've got a better idea to make this test not fail I'd be happy to change it.

Reply:
I was thinking we had other tests for when this was specifically set that would make this change to it a duplicate test but I don't actually see any so I guess this change is ok.